<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tuner_window</class>
 <widget class="QWidget" name="Tuner_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Tuner</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Images/tuner_icon.png</normaloff>Images/tuner_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
QPushButton{
background-color: rgb(255, 255, 255);
height: 50px;
width: 90px;
border:3px solid rgb(0,0,0);
border-radius:8px;
padding:10px;
}

QLabel{
	
	color: rgb(0, 255, 0);
}</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>150</y>
     <width>801</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="Sound_button" name="G_Button">
      <property name="text">
       <string>G</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="Sound_button" name="E6_Button">
      <property name="text">
       <string>E6</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="Sound_button" name="B_Button">
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="Sound_button" name="A_Button">
      <property name="text">
       <string>A</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="Sound_button" name="D_Button">
      <property name="text">
       <string>D</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="Sound_button" name="E1_Button">
      <property name="text">
       <string>E1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="graph_button">
      <property name="text">
       <string>Open_graph_window</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tuner</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Sound_button</class>
   <extends>QPushButton</extends>
   <header>Sound_button</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
